module/os/freebsd/zfs/zfs_vfsops.c

```diff
@@ -93,6 +93,10 @@ int zfs_bclone_enabled = 1;
 SYSCTL_INT(_vfs_zfs, OID_AUTO, bclone_enabled, CTLFLAG_RWTUN,
 	&zfs_bclone_enabled, 0, "Enable block cloning");
 
+int zfs_bclone_wait_dirty = 0;
+SYSCTL_INT(_vfs_zfs, OID_AUTO, bclone_wait_dirty, CTLFLAG_RWTUN,
+	&zfs_bclone_wait_dirty, 0, "Wait for dirty blocks when cloning");
```

Comment:
Could we move this into platform-independent zfs_vnops.c where it is actually used? zfs_bclone_enabled is a bad example to copy.

Reply:
My initial thought was to keep these two `zfs_bclone_*` tunables together.  Let me see about moving both.

Reply:
IIRC zfs_bclone_enabled is not global because it is used only in platform-specific code.

Reply:
It is used in the platform specific code, but it's also unnecessarily declared globally.  It looks like there's some advantage in checking this very early in the platform code, but since there's no need to declare it differently per platform I've added a commit to this PR which move it to the common code.  I think it makes sense to squash these commits, but I've kept them separate for now to make the review easier.  Thanks for taking a look.